onnxruntime/python/tools/transformers/models/whisper/whisper_chain.py

```diff
@@ -150,7 +154,18 @@ def chain_model(args):
         else [node]
     )
     beam_graph = helper.make_graph(graph_nodes, "beam-search-test", graph_inputs, graph_outputs, initializers)
-    beam_model = helper.make_model(beam_graph, producer_name="onnxruntime.transformers", opset_imports=opset_import)
+    ir_version = decoder_model.ir_version
+    if decoder_model.ir_version != encoder_model.ir_version:
+        logger.warning(
```

Comment:
assert decoder_model.ir_version == encoder_model.ir_version

Reply:
Changed to an assert